code/src/components/InputForm.js

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from 'react';
+
+const InputForm = ({
+  loading,
+  setLoading,
+  newThought,
+  setNewThought,
+  setHappyThoughtsList,
+  happyThoughtsList,
+  charCount,
+  setCharCount
+}) => {
+  const updateThoughtsList = () => {
+    setLoading(true);
+
+    fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts')
+      .then((response) => response.json())
+      .then((data) => setHappyThoughtsList(data))
+      .catch((error) => console.log(error))
+      .finally(() => {
+        console.log('it worked');
+        setLoading(false);
+        console.log('loading-Get');
+        console.log(happyThoughtsList);
+      });
+  };
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault();
+    setLoading(true);
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        message: newThought
+      })
+    };
+
+    console.log('options', options);
+    fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts', options)
+      .then((response) => response.json())
+      .then((data) => {
+        setHappyThoughtsList([data, ...happyThoughtsList]);
+        console.log(data);
+      })
+      .catch((error) => console.log(error))
+      .finally(() => {
+        setLoading(false);
+        console.log(happyThoughtsList)
+        setNewThought(''); // clear textarea
+        setCharCount(140);
+      });
+  };
+
+  useEffect(() => {
+    updateThoughtsList();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Is there a reason why you didn't use eslint-disable in the whole file? Makes it easier (in my own opinion) to remove if necessary in the top level of the file.